src/shell/shell.cpp

```diff
@@ -1092,7 +1092,7 @@ void SHELL_Init() {
 	        "\xCD\xCD\xCD\xCD\xCD\xCD\xCD\xCD\xCD\xCD\xCD\xCD\xCD\xCD\xCD\xCD\xCD\xCD\xCD\xCD\xCD\xBC\033[0m\n"
 	        "\n");
 
-	MSG_Add("SHELL_STARTUP_SUB","[color=green]dosbox-staging %s\033[0m\n");
+	MSG_Add("SHELL_STARTUP_SUB","[color=green]" CANONICAL_PROJECT_NAME " %s\033[0m\n");
```

Comment:
Suggest displaying `DOSBox Staging` as the project name here instead of `dosbox-staging`, since the former is intended to be displayed to users.

Reply:
For now, I'd like to just keep this PR with a 1:1 replacement (without these project names changing in form or how we use them). I think it's good - but should be done in it's own PR so the team can think more about them.

Reply:
If there is concern for the PR scope, perhaps you can consider undo such changes too.

Reply:
I've only de-duped the `"dosbox-staging"` string, as you commented. That's in scope - because this PR was the cause of some of that duping in the first place.

But I am reluctant to add more top-level strings for "DOSBox Staging" and further de-dupe those (because I believe doing so is more nuanced), and should have it's own PR.

Reply:
To be clear, all of the commits thus-far are what I consider in-scope.

Reply:
Well, you appeared to have enlarged the scope of the PR, which can be enlarged even further and further of course. I really wonder if there is a threshold for this.

Reply:
I've covered the items you requested against the original code. Suggest we wrap up the PR once @rderooy is able to test.

Reply:
@Wengier - I will hold off on merging the PR until you've approved; please shout if you see any remaining improvements to the current code in the PR.

Reply:
Wengier mentioned: "IMO it can be expanded to more files too, including the Windows installer script. Ideally the define for standard project name will be added too as part of cleaning up surrounding code, and IMO they can be included in the current PR too given the config.h change"

Replying: I tried this (Using Meson to populate template values in to Windows Inno setup script), and even though it's possible, it felt miss-aligned because Meson isn't a guaranteed upstream task of the Inno Installer.

It's possible to run the installer for the VisualStudio compiler results too.

Eventually I want to move to a templating system (likely Jinja), that holds all our metadata records in a top-level metadata file (project name(s), version(s), URLs, release vs. alpha, and so on).  It will be agnostic of and run prior-to the build systems, and will be fully cross platform.  This is where we can do templating into the various packaging systems, regardless of the build system being used.